src/Bundle/DependencyInjection/DunglasDoctrineJsonOdmExtension.php

```diff
@@ -45,5 +46,9 @@ public function load(array $configs, ContainerBuilder $container): void
     {
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.xml');
+
+        if (!class_exists(BackedEnumNormalizer::class)) {
```

Comment:
You should also call `removeDefinition` if `\BackedEnum` doesn't exist because the class can exist even with old PHP versions (Symfony 5.4).